### PR TITLE
sum-of-multiples: update tests to v1.5.0

### PR DESCRIPTION
**Pull request by GascaK**
Fixes exercism#1580
* Updated 3 test cases to conform to canonical data (reordered list to fit the ascending order convention)
* Updated example to check for ZeroDivisionError (anything <= 0)

### CODE REVIEW

**Comment by GascaK** [comment]
This should also Fix #1567.
Is there a way to check against the continuous integration (Travis CI) before adding an entire commit?

**Comment by cmccandless** [comment]
Travis runs `flake8` for style checks. It is available through `pip`.

File: exercises/sum-of-multiples/example.py [hunk 1]
```diff
@@ -1,4 +1,10 @@
 def sum_of_multiples(limit, multiples):
+
+    # Sanitize in-coming list
+    for i, digits in enumerate(multiples):
+        if digits <= 0:
+            del multiples[i]
+
     return sum(value for value in range(limit)
                if any(value % multiple == 0
                       for multiple in multiples))
```
Comment (cmccandless) [inline exercises/sum-of-multiples/example.py]: Might I suggest the following?

```Python
# Remove added sanitizing code
return sum(value for value in range(limit)
           if any(value % multiple == 0
                  for multiple in multiples
                  if multiple > 0))
```

File: exercises/sum-of-multiples/sum_of_multiples_test.py [hunk 2]
```diff
@@ -54,6 +54,16 @@ def test_multiples_of_1_up_to_100(self):
...
+    def test_multiples_of_0_up_to_1(self):
+        self.assertEqual(sum_of_multiples(1, [0]), 0)
+
+    def test_multiples_of_0_or_3_up_to_4(self):
+        self.assertEqual(sum_of_multiples(4, [0, 3]), 3)
+
+    def test_multiples_of_2_3_5_7_or_11_up_to_10000(self):
+        self.assertEqual(sum_of_multiples(10000,
+                         [2, 3, 5, 7, 11]), 39614537)
+
```
Comment (cmccandless) [inline exercises/sum-of-multiples/sum_of_multiples_test.py]: In https://github.com/exercism/problem-specifications/commit/8f8975109653dc798eec74f0f239ddc25af71892#diff-fbfc29c1efc68e4b7c62f38d701c6572, test case names were changed to reflect the significance of each case. Can you please update the test names here as well?

Comment (GascaK) [reply exercises/sum-of-multiples/sum_of_multiples_test.py]: No problem.

File: exercises/sum-of-multiples/example.py (hunk 1 above)
Comment (GascaK) [reply exercises/sum-of-multiples/example.py]: I'll change that no problem.

File: exercises/sum-of-multiples/example.py [hunk 3]
```diff
@@ -1,4 +1,6 @@
 def sum_of_multiples(limit, multiples):
+
```
Comment (cmccandless) [inline exercises/sum-of-multiples/example.py]: This line may be removed.

**Comment by cmccandless** [comment]
Merged; thanks for working on this!

**Pull request merged**